index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
```

Comment:
Having both the sign-in and sign-up forms simultaneously on a page is not a good design. You could have one and have a link that connects to the other. See example here https://getbootstrap.com/docs/5.3/examples/sign-in/

Reply:
I thought it was okey as I've seen many social media platforms do this, but would it be better if I have 2 seperate pages for it then? that leads to eachother?

Reply:
Got you. Make sense